pyproject.toml

```diff
@@ -11,27 +14,13 @@ write_to = "ape_hardhat/version.py"
 
 [tool.black]
 line-length = 100
-target-version = ['py36', 'py37', 'py38']
+target-version = ['py37', 'py38', 'py39']
 include = '\.pyi?$'
-exclude = '''
```

Comment:
stuff in `.gitignore` is excldued automatically (all this stuff is in `.gitignore`)

Also, it was failing with odd `black` errors only in CI. I have concluded that we don't need to be doing this here (we don't do it in core either).